pxr/imaging/rprUsd/materialRegistry.h

```diff
@@ -145,7 +147,7 @@ class RprUsdMaterialRegistry {
     std::unique_ptr<RPRMtlxLoader> m_mtlxLoader;
 #endif
 
-    std::string m_materialXStdlibPath;
+    MaterialX::DocumentPtr m_stdLibraries;
```

Comment:
Can we please load libraries during RPRMtlxLoader construction? It made code easier

Reply:
This has been discussed with Alexandr. We decided not to move lib loading to constructur because this class is a singleton. It can be initialized before other resources and it can cause incorrect lib loading